config/jobs/kubernetes/sig-cluster-lifecycle/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
Do we expect this directory to accumulate other sig-cluster-lifecycle jobs (e.g. kubeadm ones)? Should we add the sig leads here (e.g. @timothysc)?

Reply:
probably, or you can make more subdirs, your choice.

I'll probably make another PR to move all kubeadm jobs over, and I'll add people like @timothysc and @luxas to OWNERS as well.

Reply:
Works for me.